### ISSUE TITLE
hi, Gesture recognition guide for Python is not used for windows(File loading is not yet supported on Windows), i hope someone give me suggestion, thanks

### ISSUE THREAD

**Issue by xiaolinpeter** [google/mediapipe#4167]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

window 10

### MediaPipe version

0.8.10.1

### Solution

hands

### Programming Language and version

python

### Describe the actual behavior

```shell
Gesture recognition guide for Python can not used in windows 10
```

### Describe the expected behaviour

```shell
I hope Gesture recognition guide for Python can used in windows 10
```

### Standalone code/steps you may have used to try to get what you need

```shell
#!/usr/bin/env python
# -*- coding: utf-8 -*-
'''
@Project ：Air-Canvas-with-ML 
@File ：demo_2.py
@Author ：xiaolin_peter
@email：zheng.xiaolin@tslsmart.com
@Date ：2023/3/12 12:24 
'''
import mediapipe as mp
from mediapipe.tasks import python
from mediapipe.tasks.python import vision

# STEP 2: Create an ImageClassifier object.
base_options = python.BaseOptions(model_asset_path='hand_landmarker.task')
options = vision.HandLandmarkerOptions(base_options=base_options,
                                       num_hands=2)
detector = vision.HandLandmarker.create_from_options(options)

# STEP 3: Load the input image.
image = mp.Image.create_from_file("image.jpg")

# STEP 4: Detect hand landmarks from the input image.
detection_result = detector.detect(image)

# STEP 5: Process the classification result. In this case, visualize it.
# annotated_image = draw_landmarks_on_image(image.numpy_view(), detection_result)
# cv2_imshow(cv2.cvtColor(annotated_image, cv2.COLOR_RGB2BGR))
```

### Other info / Complete Logs

```shell
Traceback (most recent call last):
  File "D:/python_workspace/Air-Canvas-with-ML/pinch/demo_2.py", line 18, in <module>
    detector = vision.HandLandmarker.create_from_options(options)
  File "C:\Users\admin\.conda\envs\py385\lib\site-packages\mediapipe\tasks\python\vision\hand_landmarker.py", line 271, in create_from_options
    return cls(
  File "C:\Users\admin\.conda\envs\py385\lib\site-packages\mediapipe\tasks\python\vision\core\base_vision_task_api.py", line 65, in __init__
    self._runner = _TaskRunner.create(graph_config, packet_callback)
RuntimeError: File loading is not yet supported on Windows
```

**Comment by taaresh7** [comment]
I am facing the same issue.

**Comment by schmidt-sebastian** [comment]
We are working on this at the moment. For now, can you load the file manually and use model_asset_buffer: https://github.com/google/mediapipe/blob/master/mediapipe/tasks/python/core/base_options.py#L47

**Comment by taaresh7** [comment]
@schmidt-sebastian  Hi, is there any difference between loading the model file manually and using the new mp.solutions.hands ? I want to make sure mp.solutions.hands does not call googles api and my data is not sent to them ?

**Comment by schmidt-sebastian** [comment]
mp.solutions.hands is part of the legacy solutions, we recommend migrating to mp.tasks instead.

**Comment by lanyily** [comment]
Is this problem solved? I am facing the same issue on Windows mediapipe  0.9.0 . 
`import mediapipe as mp
BaseOptions = mp.tasks.BaseOptions
ObjectDetector = mp.tasks.vision.ObjectDetector
ObjectDetectorOptions = mp.tasks.vision.ObjectDetectorOptions
VisionRunningMode = mp.tasks.vision.RunningMode

options = ObjectDetectorOptions(
    base_options=BaseOptions(model_asset_path='efficientdet.tflite'),
    max_results=5,
    running_mode=VisionRunningMode.IMAGE)

with ObjectDetector.create_from_options(options) as detector:
    print("1")`

`Traceback (most recent call last):
  File "test.py", line 13, in <module>
    with ObjectDetector.create_from_options(options) as detector:
  File "D:\software\anaconda\lib\site-packages\mediapipe\tasks\python\vision\object_detector.py", line 211, in create_from_options
    packets_callback if options.result_callback else None)
  File "D:\software\anaconda\lib\site-packages\mediapipe\tasks\python\vision\core\base_vision_task_api.py", line 65, in __init__
    self._runner = _TaskRunner.create(graph_config, packet_callback)
RuntimeError: File loading is not yet supported on Windows`

**Comment by kuaashish** [comment]
Hello @xiaolinpeter,
This bug has been fixed in the latest release` 0.9.3.1`. You must able to load the file manually using the both `model_asset_path` as well `model_asset_buffer` as noted [here](https://github.com/google/mediapipe/releases/tag/v0.9.3.0) in the release notes. Please try out to load model using the methods and let us know if still an issue from your end. Thank you
